tests/performance-tests/src/types.d.ts

```diff
@@ -1,3 +1,7 @@
 declare module 'dotenv-json-complex' {
   export default function ({ environment: string });
 }
+
+declare module 'https://raw.githubusercontent.com/benc-uk/k6-reporter/main/dist/bundle.js' {
```

Comment:
Noice - wouldn't have known to do this.

Reply:
Yeah a lot of the k6 libs that we import like this are all in JS so unfortunately we have to duck type a bit here by declaring the types ourselves